sensor_msgs/Image bboxes
sensor_msgs/Image instance_map
std_msgs/Int8[] instance_id
sensor_msgs/PointCloud2 colored_cloud
